{{ cookiecutter.repo_name }}/Makefile

```diff
@@ -51,3 +52,9 @@ docs: ## Build docs using Sphinx and copy to docs folder (this makes it easy to
 
 ipython: ## Provides an interactive ipython prompt
 	docker exec -it $(CONTAINER_NAME) ipython
+
+setup-dvc: ## Initializes DVC in project repo
```

Comment:
Why put this in the makefile at all? Why not just call this command in the post_gen_project.sh? I feel like leaving it as a make command is inviting us to accidentally call again and blow away DVC state?

Reply:
Yeah I think we can move this to the hook. If we call it again then DVC will give an error saying that the remote already exists but this could just be part of the setup hook anyways.